src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -58,8 +58,32 @@ export function fetchInstitutions({ institutionQuery, page, onDone, onError }) {
   );
 }
 
+// Helper to clear out the facility code. Needed if the code was set via the
+// search tool and then manual address entry was selected. if this is not
+// cleared out the (incorrect) facility code will be sent along with the
+// manually entered school address.
+function removeFacilityCodeIfManualEntry(form) {
```

Comment:
Would it make sense to move this transformation to the BE with the rest of the transformations? @lihanli @jholton

Reply:
Theoretically we could make the `view:manualSchoolEntryChecked` prop a non-`view:` prop so the BE could see it and then also rip that one off as part of the processing. But it makes just as much sense to just do the work here.